src/send.py

```diff
@@ -101,6 +102,35 @@ def Send_email(txt, to_addr_str, subject):
     smtp.quit()
     return log_send_email
 
+# 获取微信access_token
+# TODO 每两小时token会过期，一天大概只能获取2000次
+def get_token():
+    payload_access_token={
+        'grant_type': configs.GRANT_TYPE,
+        'appid': configs.APPID,
+        'secret': configs.SECRET
+    }
+    token_url='https://api.weixin.qq.com/cgi-bin/token'
+    r=requests.get(token_url,params=payload_access_token)

```

Comment:
It seems that you don't handle the exception which may occur.

Reply:
Do you mean what I do just now ?